imageai:
  default_param: #param for detection with plane YOLOv3
    subscriber_id: "default"
    frame_per_second: 
    output_file_name:
    log_progress: True
    minimum_percentage_probability: #minimum percentage for recognizing objects
    return_detected_frame: False #if true, allows you to obtain the detected video frame as a Numpy array
